spec/controllers/users/registrations_controller_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Users::RegistrationsController, type: :controller do
+  render_views
+
+  let!(:user) { FactoryBot.create(:user) }
+
+  describe 'PUT #edit' do
+    
+    context 'when a skill has not been selected' do
+      
+      it 'redirects to edit page with error message' do
+        put :update
```

Comment:
This is where my test is failing. I think Devise doesn't know what resource I'm updating (User), but I'm not usre how to tell Devise this.